### DISCUSSION TITLE
Error rendering html template rollup - Update 4.4.0 -> 4.5/4.6

### DISCUSSION THREAD

**Discussion by lumsel-lt** [pretix/pretix#2440] [Q&A] [answered]
An error occured when updating pretix from version 4.3 to 4.5 and 4.6 respectively. The update step it happened recurringly was when doing the rebuild via `python -m pretix rebuild`. 

Derived from the error message the rollup script `loadConfigFile.js` was responsible. Migration stopped when rendering `webcheckin/index.html` (see error message below). Full update output was attached.

```
CommandError: An error occurred during rendering /var/pretix/venv/lib/python3.6/site-packages/pretix/plugins/webcheckin/templates/pretixplugins/webcheckin/index.html: /var/pretix/venv/lib/python3.6/site-packages/pretix/static.dist/node_prefix/node_modules/rollup/dist/shared/loadConfigFile.js:503
            : (await import(url.pathToFileURL(fileName).href)).default;
                     ^^^^^^

SyntaxError: Unexpected token import
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/var/pretix/venv/lib/python3.6/site-packages/pretix/static.dist/node_prefix/node_modules/rollup/dist/bin/rollup:22:27)
```

A complete fresh instance was set up as well (cloning from git, doing installation process as documented) with the same error. 

**System environment**

When updating to pretix version 4.5.0, this is the environment:

```
Name				Version
Ubuntu				18.04.6 LTS
Linux				4.15.0-91-generic
postgresql-10			10.18-0ubuntu0.18.04.1
nodejs				8.10.0~dfsg-2ubuntu0.4
npm				3.5.2-0ubuntu4
python3.6			3.6.9-1~18.04ubuntu1.4
pip				19.3.1
```

**Attachments**

Install output `pip3 install -U pretix==4.5.0`:
[install_pretix_4_5_0.txt](https://github.com/pretix/pretix/files/7976672/install_pretix_4_5_0.txt)

Migration output `python -m pretix migrate`:
[migrate_pretix_4_5_0.txt](https://github.com/pretix/pretix/files/7976673/migrate_pretix_4_5_0.txt)

Rebuild output `python -m pretix rebuild`:
[rebuild_pretix_4_5_0.txt](https://github.com/pretix/pretix/files/7976674/rebuild_pretix_4_5_0.txt)

**Comment by raphaelm** [comment, marked as answer]
This looks like a nodejs error and your nodejs (8.x) is veeery old. Also you use Python 3.6 which is still supported by pretix 4.5, but no longer by 4.6. I think you should get a newer Ubuntu ;)
